monitoring/api/v3/uptime-check-client/snippets.py

```diff
@@ -42,6 +42,22 @@ def create_uptime_check_config(project_name, host_name=None,
 # [END monitoring_uptime_check_create]
 
 
+# [START monitoring_uptime_check_update]
+def update_uptime_check_config(config_name, new_display_name=None,
+                               new_http_check_path=None):
+    client = monitoring_v3.UptimeCheckServiceClient()
+    config = client.get_uptime_check_config(config_name)
+    field_mask = monitoring_v3.types.FieldMask()
+    if new_display_name:
+        field_mask.paths.append('display_name')
+        config.display_name = new_display_name
+    if new_http_check_path:
+        field_mask.paths.append('http_check.path')
+        config.http_check.path = new_http_check_path
+    client.update_uptime_check_config(config, field_mask)
```

Comment:
Aside: I find it slightly annoying that the autogen client doesn't take a list of strings for `field_mask`.

Reply:
Agreed.